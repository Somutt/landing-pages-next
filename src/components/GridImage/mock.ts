import { GridImageProps } from '.';

export default {
  background: false,
  title: 'My grid',
  description:
    'Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde',
  grid: [
    {
      altText: 'Teste 1',
      imgSrc: 'https://source.unsplash.com/random/800x800?r=1',
    },
    {
      altText: 'Teste 2',
      imgSrc: 'https://source.unsplash.com/random/800x800?r=2',
    },
    {
      altText: 'Teste 3',
      imgSrc: 'https://source.unsplash.com/random/800x800?r=3',
    },
    {
      altText: 'Teste 4',
      imgSrc: 'https://source.unsplash.com/random/800x800?r=4',
    },
    {
      altText: 'Teste 5',
      imgSrc: 'https://source.unsplash.com/random/800x800?r=5',
    },
    {
      altText: 'Teste 6',
      imgSrc: 'https://source.unsplash.com/random/800x800?r=6',
    },
  ],
} as GridImageProps;
