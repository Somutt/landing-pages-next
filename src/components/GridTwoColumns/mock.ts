import { GridTwoColumnsProps } from '.';

export default {
  title: 'Grid two columns',
  text: `Lorem ipsum dolor sit amet consectetur adipisicing elit.
    Nisi dicta commodi numquam ad provident ut expedita placeat illum facilis omnis
    necessitatibus doloribus sunt beatae dolorem, maiores eos veritatis!
    Beatae, obcaecati.`,
  imgSrc: 'assets/images/backstrapi.svg',
} as GridTwoColumnsProps;
