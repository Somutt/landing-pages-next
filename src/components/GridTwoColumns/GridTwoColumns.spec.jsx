import { GridTwoColumns } from '.';

import { renderTheme } from '../../utils/renderTheme';

import mock from './mock';

describe('<GridTwoColumns />', () => {
  it('should render GridTwoColumns correctly and match snapshot', () => {
    const { container } = renderTheme(<GridTwoColumns {...mock} />);
    expect(container).toMatchSnapshot();
  });
});
