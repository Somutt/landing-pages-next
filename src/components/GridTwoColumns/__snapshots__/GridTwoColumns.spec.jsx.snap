// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GridTwoColumns /> should render GridTwoColumns correctly and match snapshot 1`] = `
.c0 {
  background: #FFFFFF;
  color: #0A1128;
  min-height: 100vh;
  display: flex;
  align-items: center;
}

.c1 {
  max-width: 120rem;
  margin: 0 auto;
  padding: 4.0rem;
  width: 100%;
}

.c5 {
  color: #0A1128;
  font-size: 3.2rem;
  text-transform: uppercase;
}

.c6 {
  font-size: 1.6rem;
}

.c2 {
  display: grid;
  grid-template-columns: 1fr 2fr;
  align-items: center;
  gap: 3.2rem;
}

.c2 .c4 {
  margin-bottom: 3.2rem;
}

@media (max-width: 768px) {
  .c2 {
    grid-template-columns: 1fr;
    text-align: center;
  }
}

@media (max-width: 768px) {
  .c3 {
    margin-bottom: 3.2rem;
  }
}

<div>
  <div
    class="c0"
    id="id-4205-856196921886"
  >
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <div
          class="c3"
        >
          <h2
            class="c4 c5"
          >
            Grid two columns
          </h2>
          <div
            class="c6"
          >
            Lorem ipsum dolor sit amet consectetur adipisicing elit.
    Nisi dicta commodi numquam ad provident ut expedita placeat illum facilis omnis
    necessitatibus doloribus sunt beatae dolorem, maiores eos veritatis!
    Beatae, obcaecati.
          </div>
        </div>
        <div
          class=""
        >
          <img
            alt="Grid two columns"
            class=""
            src="assets/images/backstrapi.svg"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
